airbyte-cdk/java/airbyte-cdk/typing-deduping/src/test/java/io/airbyte/integrations/base/destination/typing_deduping/DefaultTyperDeduperTest.java

```diff
@@ -161,12 +165,14 @@ void existingEmptyTableMatchingSchema() throws Exception {
     initialStates.forEach(initialState -> {
       when(initialState.isFinalTablePresent()).thenReturn(true);
       when(initialState.isFinalTableEmpty()).thenReturn(true);
-      when(initialState.isSchemaMismatch()).thenReturn(true);
+      when(initialState.isSchemaMismatch()).thenReturn(false);
```

Comment:
this test previously didn't actually assert anything, and was secretly broken. This is the correct version.

(specifically: if there are no final tables, isSchemaMismatch is supposed to return false)